Dockerfile

```diff
@@ -219,6 +219,10 @@ RUN bash Miniconda2-${RTD_CONDA_VERSION}-Linux-x86_64.sh -b -p /home/docs/.conda
 ENV PATH $PATH:/home/docs/.conda/bin
 RUN rm -f Miniconda2-${RTD_CONDA_VERSION}-Linux-x86_64.sh
 
+# Install Rust
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain stable
```

Comment:
We might want to specify a specific version, so that it doesn't change on rebuild, but only when we mean to update the version, if there is an easy way to specify the version.

Reply:
Yes, we can easily specify a version. I will do that now.

Reply:
Done!